.github/workflows/cypress.yml

```diff
@@ -0,0 +1,64 @@
+name: Cypress Tests
+
+on:
+    push:
+        branches: [main, develop]
+    pull_request:
+        branches: [main, develop]
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        strategy:
+            max-parallel: 4
+            matrix:
+                python-version: [3.10.13]
+
+        environment: testing
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v2
+
+            - name: create-json
+              id: create-json
+              uses: jsdaniell/create-json@1.1.2
+              with:
+                  name: "backend/malker-jam-firebase-adminsdk.json"
+                  json: ${{ secrets.FIREBASE_CREDENTIALS }}
+
+            - name: Set up .env
+              run: |
+                  echo ${{ secrets.ENV}} >> backend/.env
+
+            - name: Set up Python ${{ matrix.python-version }}
+              uses: actions/setup-python@v1
+              with:
+                  python-version: ${{ matrix.python-version }}
+
+            - name: Install Backend Dependencies
+              run: |
+                  python -m pip install --upgrade pip
+                  cd backend
+                  pip install -r requirements.txt
+
+            - name: Init backend
+              run: |
+                  cd backend
+                  python app.py &
+
+            - name: Wait for backend to be ready
+              run: sleep 10 # Ajusta el tiempo de espera según sea necesario
+
```

Comment:
I think you are missing one step:
```
- name: Set up Node
        uses: actions/setup-node@v3
        with:
          node-version: 18
          cache: 'npm'
```
That would fix:
`[BUG] Install frontend dependencies in action`